modules/configuring-firewall.adoc

```diff
@@ -38,6 +38,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |Provides {op-system-first} images
 
+|`registry.connect.redhat.com`
```

Comment:
here should be `rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`

Reply:
Addressed